### PR TITLE
Add in a totally new outlaw loadout called the "Patriot"

### PR DESCRIPTION
**Pull request by SpaceSlug2**
## About The Pull Request

This adds in a totally new and unique outlaw loadout called the "Patriot" which has absolutely no ties to a certain paramilitaristic group of genocidal maniacs.

## Why It's Good For The Game
I feel like we need more outlaw loadouts, people always want variations in their RP so I thought this would be PERFECT for those people.

## Changelog
:cl:
add: Added new things
del: Removed old things
code: changed some code
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/wasteland.dm [hunk 1]
```diff
@@ -605,6 +606,17 @@ Raider
 		/obj/item/clothing/mask/cigarette/pipe = 1,
 		/obj/item/melee/onehanded/knife/bone = 1,
 		)
+	/datum/outfit/loadout/raider_enclave
```
Comment (EgorDinamit) [inline code/modules/jobs/job_types/wasteland.dm:609]: Identation.
```suggestion
/datum/outfit/loadout/raider_enclave
```

**Comment by BadIsMyUsername** [comment]
Didn't a shitton of people who tried to play as Enclave get banned? At then playerfactions with any ties to Enclave were prohibited? And you are STILL trying to get your loadout back in, oh my god. You didn't even rename the loadout datum or the items. 

I can't tell if this is a joke or pure stupidity, but having Egor in here makes me think the latter.

**Comment by SpaceSlug2** [comment]
> Didn't a shitton of people who tried to play as Enclave get banned? And then playerfactions with any ties to Enclave were prohibited? And you are STILL trying to get your loadout back in, oh my god. You didn't even rename the loadout datum or the items.
> 
> I can't tell if this is a joke or pure stupidity, but having Egor in here makes me think the latter.

Sorry bro, but this is a totally real and true totally not to take the piss of Dawson not knowing the lore PR as I’ve spent over a year coding this

**Comment by ogrigbe** [comment]
image

**Comment by SpaceSlug2** [comment]
yes @ogrigbe

**Pull request closed without merging**